### PR TITLE
Rescale raw signals only in real_scale mode

### PR DESCRIPTION
**Pull request by jpgill86**
This PR is a response to #44 and is what I was referring to here: https://github.com/NeuralEnsemble/ephyviewer/pull/44#issuecomment-498045709

### CODE REVIEW

File: ephyviewer/traceviewer.py [hunk 1]
```diff
@@ -143,8 +143,11 @@ def compute_rescale(self):
             self.estimate_median_mad()
 
             if scale_mode=='real_scale':
-                self.viewer.params['ylim_min'] = np.nanmin(self.signals_min[self.visible_channels])
-                self.viewer.params['ylim_max'] = np.nanmax(self.signals_max[self.visible_channels])
+                if isinstance(self.viewer.source, AnalogSignalFromNeoRawIOSource):
+                    gains = self.viewer.source.neorawio.header['signal_channels']['gain']
+                    offsets = self.viewer.source.neorawio.header['signal_channels']['offset']
+                self.viewer.params['ylim_min'] = np.nanmin(self.signals_min[self.visible_channels] * gains[self.visible_channels] + offsets[self.visible_channels])
+                self.viewer.params['ylim_max'] = np.nanmax(self.signals_max[self.visible_channels] * gains[self.visible_channels] + offsets[self.visible_channels])
```
Comment (samuelgarcia) [inline ephyviewer/traceviewer.py]: This is not what I had in mind but this is very smart.
Dos it work for you ?

**Comment by samuelgarcia** [comment]
I think we can go for this.

**Comment by jpgill86** [comment]
I think that this solution is probably better for the library than #44. It fixes the scaling issue when it matters (real_scale mode) and doesn't do unnecessary scaling in other modes. Using this instead of #44 means I'll have to do a little extra work in my app, but that's OK. The library should perform well. I will merge. 😊

**Comment by jpgill86** [comment]
P.S.: Thanks for the review!

**Pull request merged**